src/lib/KeyManager.ts

```diff
@@ -308,8 +308,8 @@ class KeyManager {
   }
 
   // Verify data
-  verifyData(data: Buffer | string, signature: Buffer, withKey?: Buffer): Promise<string> {
-    return new Promise<string>(async (resolve, reject) => {
+  verifyData(data: Buffer | string, signature: Buffer, withKey?: Buffer): Promise<boolean> {
+    return new Promise<boolean>(async (resolve, reject) => {
```

Comment:
Why are we not using async await here?

Reply:
we can't do the standard async await here because kbpgp is callback based. I thought it was redundant to add the async modifier since we are returning an explicit promise but I've added it in anyway.

Reply:
Can we use the `promisify` and `callbackify` to unify what we need? https://medium.com/trabe/understanding-nodes-promisify-and-callbackify-d2b04efde0e0

Reply:
I thought we couldn't do the standard promisify pattern because kbpgp had multiple return types but it turns out promisify just takes the first one anyway so I've changed it to using this :+1: